primitives/runtime/src/transaction_validity.rs

```diff
@@ -79,6 +79,8 @@ pub enum InvalidTransaction {
 	/// A transaction with a mandatory dispatch. This is invalid; only inherent extrinsics are
 	/// allowed to have mandatory dispatches.
 	MandatoryDispatch,
+	/// The sending address is disabled or known to be invalid.
+	BadSigner,
```

Comment:
This will require that clients update before.

Reply:
:-(

Reply:
I mean it is not a hard requirement, aka we don't break authoring. We will just get a failure on `validate_block` when trying to decode the return value.